lib/json_web_token.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  class << self
+    def encode(payload, exp = 5.minutes.from_now)
+      payload[:exp] = exp.to_i
+      JWT.encode(payload, jwt_secret)
+    end
+
+    def decode(token)
+      body = JWT.decode(token, jwt_secret)[0]
```

Comment:
Likewise here, might want to specify the hashing algorithm that is used: [Verifying the cryptographic signature](https://github.com/jwt/ruby-jwt#algorithms-and-usage)